substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/DebugInfoBase.java

```diff
@@ -369,7 +369,7 @@ private TypeEntry createTypeEntry(String typeName, String fileName, Path filePat
             case INSTANCE: {
                 FileEntry fileEntry = addFileEntry(fileName, filePath);
                 typeEntry = new ClassEntry(typeName, fileEntry, size);
-                if (typeEntry.getTypeName().equals(DwarfDebugInfo.HUB_TYPE_NAME)) {
+                if (typeEntry.getTypeName().equals(DwarfDebugInfoBase.HUB_TYPE_NAME)) {
```

Comment:
This is the wrong resolution for the problem here. The constant `HUB_TYPE_NAME` should never have been located in subclass `DwarfDebugInfo` -- its only use is from the current class `DebugInfoBase`. What is needed to fix this issue is to promote this member to  `DebugInfoBase` and make it private.